go/kbfs/simplefs/simplefs_test.go

```diff
@@ -969,19 +971,38 @@ func (sr *subscriptionReporter) LastPath() string {
 	return sr.lastPath
 }
 
+func (sr *subscriptionReporter) waitForNotification(t *testing.T) {
+	t.Helper()
+	select {
+	case <-sr.lastPathNotify:
+	case <-time.After(10 * time.Millisecond):
+		t.Fatal("Timed out while waiting for notification")
+	}
+}
+
+func (sr *subscriptionReporter) requireNoNotification(t *testing.T) {
+	t.Helper()
+	select {
+	case <-sr.lastPathNotify:
+		t.Fatal("Got notification but expected none")
+	case <-time.After(10 * time.Millisecond):
+	}
+}
+
 func TestRefreshSubscription(t *testing.T) {
 	ctx := context.Background()
 	config := libkbfs.MakeTestConfigOrBust(t, "jdoe")
 	sfs := newSimpleFS(env.EmptyAppStateUpdater{}, config)
 	defer closeSimpleFS(ctx, t, sfs)
-	sr := &subscriptionReporter{Reporter: config.Reporter()}
+	sr := &subscriptionReporter{Reporter: config.Reporter(), lastPathNotify: make(chan struct{}, 1<<30)}
```

Comment:
Why is this channel so enormous?

Reply:
I need it to never block on send. `struct{}` is zero-width, so it's safe to do without any additional memory use.
https://dave.cheney.net/2014/03/25/the-empty-struct